foa_mesh/src/control.rs

```diff
@@ -0,0 +1,72 @@
+use foa::{LMacInterfaceControl, esp_wifi_hal::RxFilterBank};
+use ieee80211::mac_parser::MACAddress;
+
+use crate::{CommonResources, MeshError, rx_router::MeshRxRouterEndpoint, state::MeshState};
+
+pub struct MeshControl<'foa, 'vif> {
+    pub(crate) interface_control: &'vif LMacInterfaceControl<'foa>,
+    pub(crate) rx_router_endpoint: MeshRxRouterEndpoint<'foa, 'vif>,
+    pub(crate) common_resources: &'vif CommonResources,
+
+    pub(crate) channel: u8,
+    pub(crate) mac_address: MACAddress,
+    pub(crate) mesh_id: heapless::String<32>,
+}
+
+impl MeshControl<'_, '_> {
+    /// Set and enable all filters required for the interface.
+    fn enable_filters(&self) {
+        self.interface_control.set_filter_parameters(
+            RxFilterBank::ReceiverAddress,
+            *self.mac_address,
+            None,
+        );
+        self.interface_control
+            .set_filter_status(RxFilterBank::ReceiverAddress, true);
```

Comment:
I think it might be better to set the BSSID here anyway since it's more obvious what's actually going on here, even though the hardware technically does this automatically when none is provided. Alternatively, a comment would also work.